plugin.video.youtube/addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.youtube" name="YouTube" version="5.1.17" provider-name="bromix">
+<addon id="plugin.video.youtube" name="YouTube" version="5.2.1" provider-name="h0d3nt3uf3l, jdf76, bromix">
```

Comment:
ok no problem, thanks for the info!

Jeff


> On May 23, 2016, at 2:27 AM, Razzeee <notifications@github.com> wrote:
> 
> In plugin.video.youtube/addon.xml <https://github.com/xbmc/repo-plugins/pull/480#discussion_r64173603>:
> 
> > @@ -1,5 +1,5 @@
> >  <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
> > -<addon id="plugin.video.youtube" name="YouTube" version="5.1.17" provider-name="bromix">
> > +<addon id="plugin.video.youtube" name="YouTube" version="5.2.1" provider-name="h0d3nt3uf3l, jdf76, bromix">
> I think you will have to wait till @phil65 <https://github.com/phil65> and/or @learningit <https://github.com/learningit> can find some time. We're missing some man power in this department
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/xbmc/repo-plugins/pull/480/files/5ab7db1a24d3b6ff8e2f8a4d594407523dcf890a#r64173603>